overview.md

```diff
@@ -4,81 +4,86 @@ css_id: overview
 ---
 
 ## Overview
+A quick look at what TUF does and how it works
 
-Generally, a software update system is an application (or part of an
-application) running on a client system that obtains and installs software.
-These systems typically update the applications installed on client systems to
-introduce new features, enhancements, and security fixes.
+### Purpose, or Why Get TUF? ###
 
-Three major classes of software update systems are:
+There are literally thousands of different software update systems in common
+use today. (In fact the average Windows user has about [two
+dozen](http://secunia.com/gfx/pdf/Secunia_RSA_Software_Portfolio_Security_Exposure.pdf)
+different software updaters on their machine!)
 
-* **Application updaters** which are used by applications to update
-   themselves. For example, Firefox updates itself through its own application
-   updater.
+What these very different systems have in common is that they all identify,
+locate, and download updates for software that can add new functionalities or
+address old vulnerabilities. Software is rarely ever static, and some repositories
+receive updates on software or project metadata [every few minutes](https://ssl.engineering.nyu.edu/papers/kuppusamy_nsdi_16.pdf). This
+growing flow of updates has also created a need for better
+ways to protect the systems that manage them. Though a number of strategies have
+been introduced and used over the last decade or so to enhance the
+authenticity—and by extension, the security of update systems—most have drawbacks
+that have left update files and the repositories that host them.
+vulnerable to a number of attacks.
 
-* **Library package managers** such as those offered by many
-   programming languages for installing additional libraries. These are systems
-   such as Python's pip/easy_install + PyPI, Perl's CPAN, Ruby's RubyGems, and
-   PHP's Composer.
+TUF was launched almost a decade ago as a way to build system resilience against
+key compromises and other attacks that can spread malware or compromise a repository.
+The primary goals behind its design are:
```

Comment:
Should we mention the specification?

Reply:
As a citation or in the text? We could do it either way. It probably should be mentioned here, just not sure how

Reply:
In the text.  However, since these are design goals I'm not sure now if it makes sense to tie in the specification.

Perhaps we can toss it in the list for framework()? 

* to provide a framework (a set of libraries, file formats, **specification**, and utilities) that can be used to secure new and existing software update systems.